README.md

```diff
@@ -59,7 +59,7 @@ Flags:
   -h, --help                      Show context-sensitive help.
       --log-level="info"          Log level.
       --http-address=":7071"      Address to bind HTTP server to.
-      --node="hostname"           
```

Comment:
Ugh, this bot is removing my change to make our README.md pass... https://github.com/parca-dev/parca-agent/compare/fda48e4dfaa7f09bfac98d8998c5e4ac2f29e3b6..ff2d60ea39d5b9a4fdf7f9053cfcc1aa44ca11bd